### ISSUE TITLE
error when selector does not match anything

### ISSUE THREAD

**Issue by sstarcher** [roboll/helmfile#164]
If I run `helmfile -l name=bob sync` and we have no release named bob I would expect helmfile to exit with an error.  Currently it exists with a code of 0.

**Comment by mumoshu** [comment]
Good catch! Thats should be fixed.

**Comment by lswith** [comment]
I just encountered the opposite of this. If I have nothing that matches the selector, I would expect an error but I would also like to silence this or have a seperate error to the normal error.

Mainly I use helmfile in this way:
```
helmfile -f ./path/to/helmfile --selector state!=active delete
helmfile -f ./path/to/helmfile --selector state=active sync
```

This allows me to delete from code rather than manually. The problem is when there is nothing to delete, helmfile errors out. This should be a different error to when something is failing to delete.

**Comment by mumoshu** [comment]
@lswith Good point!
Would an exit code of `2` suffice your need?

**Comment by mumoshu** [comment]
@lswith Anyway, opened #219 for that.

**Comment by mumoshu** [comment]
I'm closing this as the original issue has been resolved via #165
